### ISSUE TITLE
add support for staffDef subelements

### ISSUE THREAD

**Issue by rettinghaus** [rism-ch/verovio#298]
For now Verovio just takes the attributes of `<staffDef>` into account, therefore following example is not rendered properly. 

``` xml
<staffDef n="3" xml:id="P3" lines="5" clef.line="2" clef.shape="G">
    <label>Tenor</label>
    <clef shape="G" line="2" dis="8" dis.place="below"/>
    <meterSig sym="common" count="4" unit="4"/>
</staffDef>
```

(Basically this is an enhancement of #283)

**Comment by lpugin** [comment]
Indeed. I looked at it a while ago and did not implemented the support for elements because it was not straight forward in some cases. I then opened this issue
https://github.com/music-encoding/music-encoding/issues/250

Generally speaking, we need to look at this in terms of priorities. Also, once elements are supported I would be quite keen to drop support for attributes because supporting both creates a significant overhead. But then I can expect issues to be open for attribute support.... ;-)
